build.sbt

```diff
@@ -2,14 +2,15 @@ import Dependencies._
 import Util._
 import com.typesafe.tools.mima.core._, ProblemFilters._
 
-def baseVersion: String = "0.1.0-M15"
+def baseVersion: String = "0.1.0-M16"
 def internalPath   = file("internal")
 
 def commonSettings: Seq[Setting[_]] = Seq(
   scalaVersion := scala211,
   // publishArtifact in packageDoc := false,
   resolvers += Resolver.typesafeIvyRepo("releases"),
   resolvers += Resolver.sonatypeRepo("snapshots"),
+  resolvers += Resolver.mavenLocal,
```

Comment:
This, and a lot of the commits in this already-merged PR, were all my temporary, local, work-in-progress trash that I intended to cleanup and not merge as is.

It's a shame this has already been merged.